taxonomy-editor-frontend/src/pages/search/index.jsx

```diff
@@ -50,18 +52,18 @@ const SearchNode = () => {
                       }}
                     fullWidth
                     onKeyDown={(e) => {
-                      if (e.key === "Enter" && !isDisabled) {
-                        navigate(`/results?query=${query}`)
+                      if (e.key === "Enter" && query.trim().length !== 0) {
+                        navigate(`/search?query=${query}`)
```

Comment:
wonder if we could solve this in another way... will try to run the stuff locally as I'm not understanding how this will look like and be used